SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -450,7 +450,7 @@
 
     <PropertyGroup Condition=" $(SonarQubeDisableRoslynCodeAnalysis) == 'true' ">
       <ErrorLog></ErrorLog>
-      <ResolvedCodeAnalysisRuleSet></ResolvedCodeAnalysisRuleSet>
+      <!-- The ResolvedCodeAnalysisRuleSet property is not changed, which means the ruleset and the analyzers are not modified for test projects. -->
```

Comment:
This is also affecting the handling of explicit exclusions. Why not just change the disable condition in SonarQubeDisableRoslynCodeAnalysis ?